tests/e2e/accounts/multisig/utils.go

```diff
@@ -0,0 +1,65 @@
+package multisig
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/protobuf/runtime/protoiface"
+
+	"cosmossdk.io/simapp"
+	"cosmossdk.io/x/bank/testutil"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var members = []sdk.AccAddress{}
+
+type ProtoMsg = protoiface.MessageV1
+
+func init() {
```

Comment:
personal preference: `init` methods should be avoided as they can not be controlled by the caller. With your `E2ETestSuite` type, it would make perfect sense for me to have this as member and use the constructor to init the them with the count passed